src/components/dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,156 @@
+import * as AlertDialogPrimitive from '@radix-ui/react-alert-dialog';
+import * as React from 'react';
+
+import clsxm from '@/lib/clsxm';
+
+import Button from '@/components/buttons/Button';
+import Typography from '@/components/typography/Typography';
+
+const AlertDialog = AlertDialogPrimitive.Root;
+
+const AlertDialogTrigger = AlertDialogPrimitive.Trigger;
+
+const AlertDialogPortal = AlertDialogPrimitive.Portal;
+
+const AlertDialogOverlay = React.forwardRef<
+  React.ElementRef<typeof AlertDialogPrimitive.Overlay>,
+  React.ComponentPropsWithoutRef<typeof AlertDialogPrimitive.Overlay>
+>(({ className, ...props }, ref) => (
+  <AlertDialogPrimitive.Overlay
+    className={clsxm(
+      'bg-gray-500/80 backdrop-blur-sm',
+      'fixed inset-0 z-40 overflow-y-auto flex items-end justify-center px-4 pt-4 pb-20 sm:p-0',
+      'data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0',
+      className,
+    )}
+    {...props}
+    ref={ref}
+  />
+));
+AlertDialogOverlay.displayName = AlertDialogPrimitive.Overlay.displayName;
+
+const AlertDialogContent = React.forwardRef<
+  React.ElementRef<typeof AlertDialogPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof AlertDialogPrimitive.Content>
+>(({ className, ...props }, ref) => (
+  <AlertDialogPortal>
+    <AlertDialogOverlay>
+      <AlertDialogPrimitive.Content
+        ref={ref}
+        onOpenAutoFocus={(e) => e.preventDefault()}
+        className={clsxm([
+          'relative self-end sm:self-center',
+          'duration-200 data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95',
+          'bg-white w-full sm:max-w-lg border shadow-xl rounded-2xl',
+          'flex flex-col gap-5 sm:gap-10 p-4 sm:p-6',
+          className,
+        ])}
+        {...props}
+      />
+    </AlertDialogOverlay>
+  </AlertDialogPortal>
+));
+AlertDialogContent.displayName = AlertDialogPrimitive.Content.displayName;
+
+const AlertDialogHeader = ({
+  className,
+  ...props
+}: React.HTMLAttributes<HTMLDivElement>) => (
+  <div className={className} {...props}>
+    {props.children}
+  </div>
+);
+AlertDialogHeader.displayName = 'AlertDialogHeader';
+
+const AlertDialogFooter = ({
+  className,
+  ...props
+}: React.HTMLAttributes<HTMLDivElement>) => (
+  <div
+    className={clsxm(
+      'border-t border-typo-divider rounded-b-2xl',
+      '-mb-4 sm:-mb-6 -mx-4 sm:-mx-6',
+      'px-4 sm:px-6 py-4',
```

Comment:
```suggestion
      'px-4 sm:px-6 py-4 sm:py-6',
```
ini kelupaan nus
![Screenshot 2024-01-26 004328](https://github.com/ppdbultimate/design-system-moon/assets/64743796/f168b3e1-7ac7-4023-9da0-2d123ef26212)

Reply:
Jadi p-4 sm:p-6 aja ya wkwkwkwk

Reply:
done